runtime/doc/options.txt

```diff
@@ -6771,80 +6771,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	by user preferences or the current theme is used.
 
 			     *'ttybuiltin'* *'tbi'* *'nottybuiltin'* *'notbi'*
-'ttybuiltin' 'tbi'	Removed. {Nvim}
+'ttybuiltin' 'tbi'	Removed. |vim-differences| {Nvim}
 
 				     *'ttyfast'* *'tf'* *'nottyfast'* *'notf'*
 'ttyfast' 'tf'		Removed. |vim-differences| {Nvim}
 
 						*'ttymouse'* *'ttym'*
-'ttymouse' 'ttym'	string	(default depends on 'term')
-			global
-			{only in Unix, doesn't work in the GUI; not
-			available when compiled without |+mouse|}
```

Comment:
This ("only on unix, not in the GUI") is a good hint that it's not an option we want to keep. 

Platform-dependent features should be implemented as providers and/or passive events, possibly with lowest-common denominator support existing in the core (as is the case with TUI mouse handling).